lib/node_modules/@stdlib/string/base/replace-before-last/README.md

```diff
@@ -0,0 +1,128 @@
+<!--
+
+@license Apache-2.0
+
+Copyright (c) 2024 The Stdlib Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# replaceBeforeLast
+
+> Replace the substring before the last occurrence of a specified search string.
+
+<!-- Section to include introductory text. Make sure to keep an empty line after the intro `section` element and another before the `/section` close. -->
+
+<section class="intro">
+
+</section>
+
+<!-- /.intro -->
+
+<!-- Package usage documentation. -->
+
+<section class="usage">
+
+## Usage
+
+```javascript
+var replaceBeforeLast = require( '@stdlib/string/base/replace-before-last' );
+```
+
+#### replaceBeforeLast( str, search, replacement, fromIndex )
+
+Replaces the substring before the last occurrence of a specified search string.
+
+```javascript
+var str = 'beep boop';
+var out = replaceBeforeLast( str, ' ', 'loop', str.length );
+// returns 'loop boop'
+
+out = replaceBeforeLast( str, 'o', 'bar', str.length );
+// returns 'barop'
+```
+
+</section>
+
+<!-- /.usage -->
+
+<!-- Package usage notes. Make sure to keep an empty line after the `section` element and another before the `/section` close. -->
+
+<section class="notes">
+
+## Notes
+
+-   If a search string is not present in a provided string, the function returns the provided string unchanged.
+-   If a search string is an empty string, the function returns the provided string unchanged.
+-   If `fromIndex` is less than `0` or greater than the `search` string length, the function returns the provided string unchanged.
```

Comment:
Same comment as in https://github.com/stdlib-js/stdlib/pull/1365/files#r1501925656.